projects/ccd-case-ui-toolkit/src/lib/shared/domain/definition/display-context-enum.model.ts

```diff
@@ -6,5 +6,6 @@ export enum DisplayContextParameter {
 
 export enum DisplayContextCustomParameter {
   CaseFileView = 'CaseFileView',
-  LinkedCases = 'LinkedCases'
+  LinkedCases = 'LinkedCases',
+  QueryManagement = 'QueryManagement'
```

Comment:
Why is this needed as display context?

Reply:
This DisplayContextCustomParameter is used in Palette Service's componentLauncherRegistry.